setup.py

```diff
@@ -6,7 +6,7 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-VERSION = '0.3.25'
+VERSION = '0.3.26'
```

Comment:
Version should not be updated in the PR, there is a process in the wiki described how to update a version